### DISCUSSION TITLE
Mesh refinement

### DISCUSSION THREAD

**Discussion by PEI0214** [idaholab/moose#28694] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hello，
I refined part of the mesh. Apply pressure to the top. The results looked a little strange. The refined mesh seems to be separated from the unrefined mesh. What causes this? Are there any good suggestions for this problem?
The initial mesh, the refined mesh and the results are as follows:
<details>

<summary>mesh and result</summary>

### initial mesh

![image](https://github.com/user-attachments/assets/a18cbe1c-14c8-43fa-a57e-bd015cc68178)

### refined mesh

![image](https://github.com/user-attachments/assets/7f67c7df-67c7-4911-947b-861cccb2338e)

### result

![image](https://github.com/user-attachments/assets/c7ef33b9-67a2-4283-8d94-39001d829cae)

</details>

**Comment by GiudGiud** [comment]
Hello

your initial mesh is not attached (not conformal). It does not improve with refinement. 
You can see it where you drew your first red circle in the first page

**Reply by PEI0214** [reply]
The initial mesh is the first diagram. This is the mesh refined from the upper part of the initial mesh.

**Reply by GiudGiud** [reply]
I know

> The refined mesh seems to be separated from the unrefined mesh

it was already separated in the initial mesh.
I dont know what you used for mesh generation but it did not attach the top and bottom parts. Otherwise on the left and right of the first pictures we would see nodes being shared by both parrts

**Reply by PEI0214** [reply]
The initial mesh is not separated. The first image looks separated because of the Angle of the screenshot. It actually looks like this:
![9ad16b9ed8a7b7021c7851e27c44a999](https://github.com/user-attachments/assets/53acf7b4-de86-43f4-8842-e6389297e6bb)
![af36e070f37262960eb96a765984b146](https://github.com/user-attachments/assets/2bf4a510-a858-4b22-9548-ab43316a639a)

**Reply by GiudGiud** [reply]
what do the mesh diagnostics say?
https://mooseframework.inl.gov/source/meshgenerators/MeshDiagnosticsGenerator.html

**Reply by PEI0214** [reply]
Thank you for your answer. I figured out what the problem was.
